### PR TITLE
improve the Module Source documentation

### PR DESCRIPTION
**Pull request by afeld**
* Adds cross-links within the Module Source page, which should help with navigation.
* Information about how to use modules from the Registry was scattered across a few places, and only fully documented in a somewhat buried page. This pull request removes the other examples, to point to that canonical place.
* Some spelling fixes.

Thanks!

### CODE REVIEW

File: website/docs/modules/sources.html.markdown [hunk 1]
```diff
@@ -44,24 +44,7 @@ Updates for file paths are automatic: when "downloading" the module using the [g
...
-}
-```
-
-The above example would use the
-[Consul module for AWS](https://registry.terraform.io/modules/hashicorp/consul/aws)
-from the public registry.
-
-You can learn more about the registry at the
-[Terraform Registry documentation section](/docs/registry/index.html).
+See the [Terraform Registry documentation](/docs/registry/modules/use.html#using-modules).
```
Comment (apparentlymart) [inline website/docs/modules/sources.html.markdown]: I'd like to still retain the content here, even though it is indeed a bit redundant with what you linked to, as an example to help users who are just skimming this page to understand the meaning of some code they're looking at. If a user sees `source = "hashicorp/consul/aws"` in some existing config then they may come here to learn what kind of module reference that is.

Comment (afeld) [reply website/docs/modules/sources.html.markdown]: Done!

**Comment by afeld** [comment]
@apparentlymart Ready to go!

**Comment by apparentlymart** [comment]
Thanks for working on this @afeld, and sorry for the delay in getting it merged. I'm going to get it in now!

**Pull request merged**